aplib.net-demo/Assets/Scripts/SharedRandom.cs

```diff
@@ -35,6 +36,7 @@ private static void Assign()
             }
 
             _local = new Random(_seed);
+            UnityRandom.seed = _seed;
```

Comment:
I am getting a deprecation warning: "'Random.seed' is obsolete: 'Deprecated. Use InitState() function or Random.state property instead.'CS0618'"

Is it possible to change?